### PR TITLE
Create rre

### PR DESCRIPTION
**Pull request by THX-21**
create rre

### CODE REVIEW

File: rre [hunk 1]
```diff
@@ -0,0 +1 @@
+add rre
```
Comment (THX211) [inline rre]: change it

**Comment by THX211** [comment]
change

File: rre
Comment (THX211) [inline rre:1]: change something

Comment (THX211) [inline rre:1]: Duplicate of #

**Comment by THX-21** [comment]
this is better than #1

File: rre [hunk 2]
```diff
@@ -0,0 +1 @@
+add 11111111111111111111111111111111111111111111111
```
Comment (THX211) [inline rre:1]: good

**Comment by THX-21** [comment]
good 8c0c0cfc2e9b2ab77fe0fb4d8a79ee8c88325935

**Comment by THX-21** [comment]
- [x] write the code
- [x] log the code
- [ ] change the code

**Comment by THX211** [comment]
> create rre

how much?

**Comment by THX211** [comment]
:bug: to 💀

**Pull request closed without merging**